main.cpp

```diff
@@ -526,6 +561,45 @@ void Body::alarmOverDust(float time)
     ++time;
 }
 
+int Knob::setValue(int pval, int cval)
+{
+
+    Knob::Led led;
+    led.Num = pval;
+
+    float step = 1.0f / ( static_cast<float>(cval) + 1.0f );
+    
+    if ( pval < cval )
+    {
+        led.Brightness = 0;
+
+        for ( led.Num = 0; led.Num <= cval; ++led.Num )
+        {
+            led.Brightness+=step;
+            led.set();
+        }
+    }
+    else
+    {
+        while ( led.Num > cval )
+        {
+            led.Brightness = 0;
+            led.set();
+            --led.Num;
+        }
+
+        ( cval !=  0  ? led.Brightness = 1 : 0 );

```

Comment:
When you write ternary expressions, you can make them more readable by writing them like this:
```
led.brightness = (expr ? true statement : false statement);
```
as in:
```
led.brightness = (cval != 0 ? 1 : 0 );
```
- [ ] revise
- [ ] `Brightness` is a member variable, and member variable names need to start with a `lowerCaseLetter`

Reply:
does it work with "int"?